.github/workflows/image-push.yaml

```diff
@@ -0,0 +1,57 @@
+name: Push container image
+on: 
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*.*.*'
+env:
+  image-push-owner: 'AlonaKaplan'
```

Comment:
Where is this `env` used?

Reply:
Maybe something internally
I see it on other refs so copied the template
i.e
https://github.com/k8snetworkplumbingwg/whereabouts/blob/aec29ad859d08ac696b2be17ee7db398b2b0141a/.github/workflows/image-push-release.yml#L7